source/braille.py

```diff
@@ -1441,11 +1441,150 @@ def _setCursor(self, info: textInfos.TextInfo):
 class ReviewTextInfoRegion(TextInfoRegion):
 
 	allowPageTurns=False
+	_currentSelection: textInfos.TextInfo | None = None
+	_currentScriptName: str | None = None
+	_withinSelection: bool = False
+
+	def _getSelection(self) -> textInfos.TextInfo:
+		"""Gets selection for use in update function.
+		:return: selection, reading unit from it, or review position when
+		there is no selection or showing selection is disabled
+		"""
+		self._withinSelection = False
+		if not config.conf["braille"]["showSelection"]:
+			self._currentSelection = None
+			return api.getReviewPosition().copy()
+		info: textInfos.TextInfo
+		try:
+			info = self.obj.makeTextInfo(textInfos.POSITION_SELECTION)
+		except (LookupError, RuntimeError):
+			self._currentSelection = None
+			info = api.getReviewPosition().copy()
+			# Info should be collapsed, but it is not always, at least when

```

Comment:
For reason I do not know when "show selection" is enabled without this false selections were shown. This blocks them. When "show selection" is disabled - this is not necessary because selection is not shown in braille, and behavior is same as it is now.